src/lib/components/Footer/FooterLink.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames';
+import { FC, PropsWithChildren } from 'react';
+
+export type FooterLinkProps = PropsWithChildren<{
+  href: string;
+  className?: string;
+  ariaLabel?: string;
+}>;
+
+export const FooterLink: FC<FooterLinkProps> = ({ href, children, ariaLabel, className }) => {
+  return (
+    <li className={classNames('mr-4 last:mr-0 md:mr-6', className)}>
+      <a href={href} aria-label={ariaLabel} className="hover:underline">
```

Comment:
Screen readers can derive an accessible name (what gets read aloud to users) from context. For example, the body (children) of the anchor will already be selected by screen readers, and aria-label is not needed.

Reply:
@tulup-conner thanks for the feedbacks :)